lib/src/footer/types.ts

```diff
@@ -0,0 +1,71 @@
+type Space = "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | "xxlarge";
+type Size = {
+  top?: Space;
+  bottom?: Space;
+  left?: Space;
+  right?: Space;
+};
+type Padding = {
+  top?: Space;
+  bottom?: Space;
+  left?: Space;
+  right?: Space;
+};
+type SocialLink = {
+  logo?: ImgElement;
+  logoSrc?: string;
+  href?: string;
+};
+type BottomLink = {
+  text?: string;
+  href?: string;
+};
+type ImgElement = HTMLElement & (SVGElement | HTMLImageElement);
+
+type FooterPropsType = {
+  /**
+   * An array of objects representing the links that will be rendered as
+   * icons at the top-right side of the footer. Each object has the
+   * following properties:
+   * - logo: Element used as the icon for the link.
+   * - logoSrc: The path of an icon for the link. @deprecated
+   * - href: URL of the page the link goes to.
+   */
+  socialLinks?: SocialLink[];
+  /**
+   * An array of objects representing the links that will be rendered at
+   * the bottom part of the footer. Each object has the following
+   * properties:
+   * - text: Text for the link.
+   * - href: URL of the page the link goes to.
+   */
+  bottomLinks?: BottomLink[];
+  /**
+   * The center section of the footer. Can be used to render custom
+   * content in this area.
+   */
+  children: React.ReactNode;
+  /**
+   * The text that will be displayed as copyright disclaimer.
+   */
+  copyright?: string;
+  /**
+   * Value of the tabindex for all interactuable elements, except those
+   * inside the custom area.
+   */
+  tabIndex?: number;
+  /**
+   * Size of the margin to be applied to the component ('xxsmall' |
+   * 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'xxlarge').
```

Comment:
Try to remove the values from the comments since they are already defined in the type.

Reply:
Done